.github/workflows/vscode.yml

```diff
@@ -14,30 +14,38 @@ on:
 jobs:
   test:
     strategy:
+      fail-fast: false
       matrix:
         # TODO: support other platform such as macos
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest]
         ruby: ['3.1.0']
     runs-on: ${{ matrix.os }}
     steps:
     - name: Checkout
       uses: actions/checkout@v3
     - name: Set up Ruby
+      id: setup-ruby
```

Comment:
Why do we need to set `id` here?

Reply:
Ah, it's for `steps.setup-ruby.outputs.ruby-prefix`. I just copy-and-pasted the file from #103. 😅 

```yaml
env:
  RUBY_BIN_PATH: ${{ runner.os == 'macOS' && format('{0}/{1}', steps.setup-ruby.outputs.ruby-prefix, 'bin') || '' }}
```

https://github.com/ruby/vscode-typeprof/pull/103/files#diff-731c6f95e51cf7fd242dc335a02c96b18fd955089cfcfdb5c32cad3e884e02ecR49

Should I remove it?

Reply:
Ah, that's fine.